content/reference/views/index.md

```diff
@@ -212,3 +212,69 @@ module.exports = function (state, emit) {
    showing the same title we already have, then we don't need to do anything.
    Conditional statements are less instructions on the CPU than function calls,
    so this provides us with a little speedup (and less noise).
+
+## Composable Views
+As applications grow, you'll probably find that there will be plenty of views
+that share a lot of the same layout. If you find that you're repeating the same
+layout in a lot of code, it can be beneficial to make it reusable instead.
+
+The most common way to create reusable templates, is to create a function that
+takes a view as an argument and returns a view. Inside the function the
+childView is called, and wrapped with some HTML. The end result is a nicely
+composed function that's also known as a "composable view", "higher order view",
+"template".
+
```

Comment:
I would rewrite this entire paragraph:

The easiest way to create a reusable view in Choo is to create a function that takes a view (meaning a snippet of HTML) and then inserts it into another, function-internal view. This makes it easy to create functions that construct HTML that you always want to keep static, such as a menu bar, but at the same time construct dynamic HTML defined by whatever you pass it as an argument. Functions like these are called "composable functions" and, funnily enough, are a fundamental piece of functional programming!

or something like that

Reply:
`function-internal` is an absolutely stupid phrase as i just noticed